tools/packaging/kata-deploy/local-build/kata-deploy-build-and-upload-payload.sh

```diff
@@ -18,14 +18,23 @@ pushd ${KATA_DEPLOY_DIR}
 IMAGE_TAG="${REGISTRY}:kata-containers-$(git rev-parse HEAD)-$(uname -m)"
 
 echo "Building the image"
-if [ "$(uname -m)" = "s390x" ]; then
-	docker build \
-		--build-arg IMG_NAME=clefos \
-		--build-arg IMG_TAG=7 \
-		--tag ${IMAGE_TAG} .
-else
-	docker build --tag ${IMAGE_TAG} .
-fi
+case $(uname -m) in
```

Comment:
I find it very confusing that, both in the old and new code, we use *both* `IMG_TAG` and `IMAGE_TAG`. Given that it looks like the script is indeed using `IMAGE_TAG`, that reinforce my suggestion to name the dockerfile ARG as `IMAGE_TAG` as well, no?

Reply:
IMAGE_TAG is the tag you want your built image to receive, for instance, `kata-containers-x86_64`.
IMG_TAG, inside the script, is the `7` in case of `centos:7`.

Reply:
Oh, I totally misinterpreted this. Hmm, maybe this suggest we should have a "source" and "destination" prefix to some of those? Or are you comfortable enough that the code is maintainable as is?

Reply:
SRc / DEST is also not the most clear or accurate option.
Let me iterate on this one and I'll count on you for a second pass to make sure we can make it cleaner / easier to maintain.

Reply:
Thanks. It can easily be deferred if the change as is unblocks something else.

Reply:
I've changed the IMG_NAME and IMG_TAG to BASE_IMAGE_NAME and BASE_IMAGE_TAG, which makes it easier to understand that those are related to the container used as the basis for the kata-deploy daemonset.

IMAGE_TAG was kept as it is, as it was already like that, as this patch set is not directly touching it, and as I do believe the changes mentioned above make it clear enough to avoid any confusion.

Let me know what you think.